Products/Zuul/catalog/model_catalog_tool.py

```diff
@@ -270,7 +270,7 @@ def getValue(obj):
         return sorted((unbrain(brain) for brain in queryResults),
                       key=getValue, reverse=reverse, cmp=natural_compare)
 
-    def search(self, types=(), start=0, limit=None, orderby='name',
+    def search(self, types=(), start=0, cursor="*", limit=None, orderby='modelindex_uid',
```

Comment:
Can you use the constant name from modelindex rather than hard-coding the initial cursor mark?  This is for both `cursor` and `orderby`.